Assets/FrostedGlass/Scripts/CommandBufferBlur.cs

```diff
@@ -62,6 +63,13 @@ void Initialize()
 
         _Camera = GetComponent<Camera>();
 
+        if (_Camera.allowHDR && SystemInfo.SupportsRenderTextureFormat(RenderTextureFormat.DefaultHDR))
+        {
+            _BufferRenderTextureFormat = RenderTextureFormat.DefaultHDR;
+        } else {
```

Comment:
Remove this

Reply:
do you mean remove the part *below* that, since it would be the default once the previous change is done? If I remove the part you highlighted, it would stop working.

Reply:
This part :

else {
      _BufferRenderTextureFormat = RenderTextureFormat.ARGB32;
}